lib/src/ovis_util/big_dstring.h

```diff
@@ -104,6 +104,8 @@ static inline const char *bdstrval(const big_dstring_t *dsPtr); \
 static void bdstr_free(big_dstring_t *dsPtr) \
 { \
 	assert(NULL != dsPtr); \
```

Comment:
Given the proposed change, callers to bdstr_free() may now assume that dsPtr may be NULL. If we add this assert, these apps will break.